src/control/server/config_test.go

```diff
@@ -322,15 +333,62 @@ func TestConfigValidation(t *testing.T) {
 			// Apply extra config test case
 			config = tt.extraConfig(config)
 
-			err = config.Validate()
-			if tt.errMsg != "" {
-				ExpectError(t, err, tt.errMsg, name)
-				return
+			CmpErr(t, tt.expErr, config.Validate(log))
+		})
+	}
+}
+
+func TestConfig_RelativeWorkingPath(t *testing.T) {
+	for name, tt := range map[string]struct {
+		inPath    string
+		expErrMsg string
+	}{
+		"path exists":       {inPath: "uncommentedDefault"},
+		"path doesnt exist": {expErrMsg: "no such file or directory"},
+	} {
+		t.Run(name, func(t *testing.T) {
+			testDir, err := ioutil.TempDir("", strings.Replace(t.Name(), "/", "-", -1))
```

Comment:
For the record, there is a helper for this now: https://godoc.org/github.com/daos-stack/daos/src/control/common#CreateTestDir